js/app/services/notesmodel.js

```diff
@@ -27,6 +27,10 @@ app.factory('NotesModel', function () {
             return this.notes;
         },
         get: function (id) {
+            if(this.notesIds[id].error) {
+                OCdialogs.alert(this.notesIds[id].errorMessage,'Error!');
```

Comment:
This leads to a warning in gulp: `'OCdialogs' is not defined.`

Why do you aren't using the official Nextcloud way (using `OC.Notification` like in `appcontroller.js`) to present the error?

Reply:
OCDialogs is in nextcloud core, but I feel like perhaps OC.dialogs is what is intended for apps to use. I think they both map to the same object. I'll update the reference.

OC.dialogs produces file error messages consistent with the file error messages in nextcloud core, so in my opinion this method is preferable